src/getConfig.js

```diff
@@ -38,6 +51,43 @@ function isSimple(config) {
 }
 
 /**
+ * Custom jest-validate reporter for unknown options
+ * @param config
+ * @param example
+ * @param option
+ * @param options
+ * @returns {void}
+ */
+function unknownValidationReporter(config, example, option, options) {
+  /**
+   * If the unkonwn property is a glob this is probably
+   * a typical mistake of mixing simple and advanced configs
+   */
+  if (isGlob(option)) {
+    const message = `
+  Unknown option ${chalk.bold(`"${option}"`)} with value ${chalk.bold(
+      format(config[option], { inlineCharacterLimit: Number.POSITIVE_INFINITY })
+    )} was found in the config root.
+  
+  You probably trying to mix simple and advanced config formats. 
+  
+  Adding 
+  
+  ${chalk.bold(`"linters": {
+    "${option}": ${format(config[option], { inlineCharacterLimit: Number.POSITIVE_INFINITY }, '  ')}
+  }`)}
+   
+  will fix it and remove this message.`
+
+    const comment = options.comment
+    const name = (options.title && options.title.warning) || 'WARNING'
+    logValidationWarning(name, message, comment)
```

Comment:
Should there be a return after this? I haven't used `jest-validate` before so not too sure about this.

Reply:
Yes! Added. Thanks